FIrstModule/FIrstModule/Program.cs

```diff
@@ -1,63 +1,212 @@
-    string item = Console.ReadLine().ToLower();
-    if (item != null)
-    {
-
-    }
-    else Console.WriteLine("String is null.");
-}
-void RemoveItem(string[,] items)
-{
-
-}
-void MarkAs(string[,] items)
-{
-
-}
-void Show(string[,] items)
-{
-
-}
-
-
+﻿
 Console.WriteLine("Hello, input the command for your tasks.");
-
+List<string> items = new List<string>() { "str", "add-item", "make a homework" };
 while (true)
 {
+    Console.WriteLine(Environment.NewLine);
     Console.WriteLine(">add-item");
     Console.WriteLine(">remove-item");
     Console.WriteLine(">mark-as");
     Console.WriteLine(">show");
     Console.WriteLine(">exit");
-
     Console.WriteLine("Input a command:");
     var command = Console.ReadLine();
-
+    Console.WriteLine(Environment.NewLine);
     switch (command)
     {
         case "add-item":
             {
-                AddItem(items);
-                break;
+                Console.WriteLine("Input a task:");
+                var task = Console.ReadLine();
+
+                if (!string.IsNullOrEmpty(task))
+                {
+                    if (!items.Any())
+                    {
+                        items.Add(task); break;
+                    }
+                    bool IsTask = true;
+                    foreach (string item in items)
+                    {
+                        if (task.ToLower().Contains(item.ToLower()))
+                        {
+                            IsTask = false;
+                            break;
+                        }
+                    }
+                    if (!IsTask)
+                    {
+                        Console.WriteLine(Environment.NewLine);
+                        Console.WriteLine("This task is already"); break;
+                    }
+                    else { items.Add(task); break; }
+                }
+                else
+                {
+                    Console.WriteLine(Environment.NewLine);
+                    Console.WriteLine("Error.. You input anything! Try again"); break;
+                }
             }
         case "remove-item":
             {
-                RemoveItem(items); break;
+                if (!items.Any())
+                {
+                    Console.WriteLine(Environment.NewLine);
+                    Console.WriteLine("There aren`t tasks"); break;
+                }
+                foreach (string item in items)
+                {
+                    Console.WriteLine(item);
+                }
+                Console.WriteLine(Environment.NewLine);
+                Console.WriteLine("What you want to delete?");
+                var delete = Console.ReadLine();
+                if (!string.IsNullOrEmpty(delete))
+                {
+                    if (delete == "*")
+                    {
+                        items.Clear(); break;
+                    }
+                    bool IsTask = true;
+                    foreach (string item in items)
```

Comment:
Don't you think that you repeat that code to find an item? If you repeat some code, shouldn't it be moved to some reusable method?